bindings/python/src/OpenSpaceToolkitPhysicsPy/Time/Instant.cpp

```diff
@@ -235,5 +245,26 @@ inline void OpenSpaceToolkitPhysicsPy_Time_Instant(pybind11::module &aModule)
             )doc"
         )
 
+        .def_static(
+            "parse",
+            &Instant::Parse,
+            arg("string"),
+            arg_v("scale", DEFAULT_TIME_SCALE, "Scale.UTC"),
```

Comment:
Should we allow an implicit timescale? (it's usually the source of many errors?)

Reply:
Nah good catch. This would be the only constructor with a default Scale... and it's probably not a good idea. Gonna remove.